realm-jni/src/io_realm_internal_UncheckedRow.cpp

```diff
@@ -288,3 +288,10 @@ JNIEXPORT jboolean JNICALL Java_io_realm_internal_UncheckedRow_nativeIsAttached
     TR_ENTER_PTR(nativeRowPtr)
     return ROW(nativeRowPtr)->is_attached();
 }
+
+JNIEXPORT jboolean JNICALL Java_io_realm_internal_UncheckedRow_nativeHasField
```

Comment:
Isn't the name inconsistent with the other methods? I would prefer it was called `Java_io_realm_internal_IncheckRow_nativeHasColumn`.